README.rst

```diff
@@ -108,6 +103,36 @@ or::
 if your compiler does not support gcc-style options, if you are cross
 compiling, etc.
 
+Additional Language Interfaces
+----------------------------------------
+
+The Fortran interface is built alongside the library automatically.
+
+Python users can install using::
+
+    pip install libceed
+
+or in a clone of the repository via ``pip install .``.
+
+Julia users can install using::
+
+   $ julia
+   julia> ]
+   pkg> add LibCEED
+
+in the Julia package manager or in a clone of the repository via::
+
+    JULIA_LIBCEED_LIB=/path/to/libceed.so julia
+    julia> # press ] to enter package manager
+    (env) pkg> build LibCEED
```

Comment:
I know this does not pertain this PR in particular, but I feel that the implicit dependencies needed for the `LibCEED.jl` Julia interface libCEED should be documented. 
For instance, when I tried to build it, I got the following 
```
(@v1.5) pkg> add LibCEED
  Resolving package versions...
ERROR: Unsatisfiable requirements detected for package CEnum [fa961155]:
 CEnum [fa961155] log:
 ├─possible versions are: [0.1.0-0.1.2, 0.2.0, 0.3.0, 0.4.0-0.4.1] or uninstalled
 ├─restricted to versions * by an explicit requirement, leaving only versions [0.1.0-0.1.2, 0.2.0, 0.3.0, 0.4.0-0.4.1]
 ├─restricted by compatibility requirements with LibCEED [2cd74e05] to versions: 0.4.0-0.4.1
 │ └─LibCEED [2cd74e05] log:
 │   ├─possible versions are: 0.1.0 or uninstalled
 │   └─restricted to versions * by an explicit requirement, leaving only versions 0.1.0
 └─restricted by compatibility requirements with CuArrays [3a865a2d] to versions: 0.2.0 — no versions left
```

I do see the compatibility requirements in [Project.toml](https://github.com/CEED/libCEED/blob/main/julia/LibCEED.jl/Project.toml) file, but I couldn't find this anywhere else in the docs. Should we rely on users having to go read the file themselves?

Reply:
@pazner is best suited to answer this